back/app/models/idea.rb

```diff
@@ -136,6 +136,9 @@ class Idea < ApplicationRecord
   has_many :embeddings_similarities, as: :embeddable, dependent: :destroy
   has_many :authoring_assistance_responses, dependent: :destroy
 
+  has_many :related_idea_associations, class_name: 'RelatedIdea', dependent: :destroy
```

Comment:
Maybe we could call the model of the join table `IdeaRelation` instead of `RelatedIdea`? Would make this more elegant, and it's also a bit more accurate.

Reply:
Good idea! I'll rework that.